pkgs/top-level/all-packages.nix

```diff
@@ -6443,6 +6443,11 @@ in
 
   beecrypt = callPackage ../development/libraries/beecrypt { };
 
+  beignet = callPackage ../development/libraries/beignet {
+    inherit (pkgs) llvmPackages; 
```

Comment:
There's no need to inherit here, `callPackage` should fill in any missing parameters.